yt/frontends/ramses/fields.py

```diff
@@ -92,6 +96,18 @@ def _temperature(field, data):
         self.add_field(("gas", "temperature"), sampling_type="cell",  function=_temperature,
                         units=self.ds.unit_system["temperature"])
         self.create_cooling_fields()
+        # See if we need to load the rt fields
+        foldername  = os.path.abspath(os.path.dirname(self.ds.parameter_filename))
+        rt_flag = not os.system('ls ' + foldername + '/info_rt_*.txt 1>/dev/null 2>/dev/null')
```

Comment:
I'm going to change this to use `os.path.exists` since this won't work on systems that don't have `ls` (e.g. Windows).